noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/view/screen/SignUpScreen.kt

```diff
@@ -53,15 +55,16 @@ fun SignUpScreen(
     viewModel: RegisterViewModel
 ) {
 
-    val viewState = viewModel.authFlow.collectAsState(initial = null).value
-
-    when (viewState) {
+    when (val viewState = viewModel.authFlow.collectAsState(initial = null).value) {
```

Comment:
Same as above ⬆️

Reply:
@PatilShreyas I believe, thats an instance of view state, and its only being used in when scope, and creation of it above when, increases the scope and will allow mutation of state outside  the when statement,  but for now there is no usage of view state outside of it, thus restrictes its mutation, let me know if it feels like a good enough reason to keep it as is, or I can revert it, otherwise.

Reply:
Yeah please revert it. It looks more readable.